daemon/logger/copier.go

```diff
@@ -69,3 +73,8 @@ func (c *Copier) copySrc(name string, src io.Reader) {
 func (c *Copier) Wait() {
 	c.copyJobs.Wait()
 }
+
+// Closed closed the copier
+func (c *Copier) Close() {
+	c.closed = true
```

Comment:
yes, which is why we wait for <-exit after Close() is called. I think that should suffice ?